src/Pool.ts

```diff
@@ -11,6 +12,10 @@ export class Pool extends Entity {
     super(_root, ['pool', id])
   }
 
+  reports() {
+    return new Reports(this._root, this.id)
+  }
```

Comment:
I think it should either be a query or a property. Property makes the most sense I think. `await pool.reports.balanceSheet()`